lighthouse-logger/index.js

```diff
@@ -207,5 +219,10 @@ class Log {
 }
 
 Log.events = new Emitter();
+Log.getEntries = _ => {
```

Comment:
I realize this (appears) to be going for the same `getEntries()` name as the `Performance.getEntries()`, but it's definitely confusing in the context of a logger (where you might expect it to return all log entries).

WDYT about `Log.getTimeEntries()`?

Reply:
I had the same feeling as well, will do 👍

Reply:
settled on takeTimeEntries to mimic the semantics and name of PerfObserver 👍